site/profile/manifests/users.pp

```diff
@@ -171,4 +172,13 @@
     line    => "${name} ALL=(ALL) NOPASSWD:ALL",
     require => File['/etc/sudoers.d/90-puppet-users'],
   }
+
+  if $authenticationmethods != '' {
+    sshd_config { "${name} authenticationmethods":
+      ensure    => present,
+      condition => "User ${name}",
+      key       => 'AuthenticationMethods',
+      value     => "${authenticationmethods}"
```

Comment:
`value => $authenticationmethods`

Reply:
My bad, I was the one who put braces in the sshd_config I shared.

Reply:
ok, that fixes it